frontend/src/Settings/Metadata/MetadataProvider/MetadataProvider.js

```diff
@@ -54,6 +61,37 @@ function MetadataProvider(props) {
                 </FormGroup>
               </FieldSet>
             }
+
+            <FieldSet legend="Write Metadata to Audio Files">
+              <FormGroup>
+                <FormLabel>Tag Audio Files with Metadata</FormLabel>
+
+                <FormInputGroup
+                  type={inputTypes.SELECT}
+                  name="writeAudioTags"
+                  helpTextWarning="Selecting 'All files' will alter existing files when they are imported."
+                  helpLink="https://github.com/Lidarr/Lidarr/wiki/Write-Tags"
+                  values={writeAudioTagOptions}
+                  onChange={onInputChange}
+                  {...settings.writeAudioTags}
+                />
+              </FormGroup>
+
+              {
+                settings.writeAudioTags.value !== 'no' &&
+                  <FormGroup>
+                    <FormLabel>Scrub Existing Metadata from Audio Files</FormLabel>
```

Comment:
Can we shorten up this label to something like `Scrub Existing Tags`, currently it wraps in UI. Help text will explain more if user needs it.

![image](https://user-images.githubusercontent.com/376117/54065269-51161600-41ec-11e9-8022-ffee9dc2232d.png)